core/state_processor.go

```diff
@@ -659,7 +645,7 @@ func (p *StateProcessor) Process(block *types.WorkObject, batch ethdb.Batch) (ty
 						if err != nil {
 							return nil, nil, nil, nil, 0, 0, 0, nil, nil, fmt.Errorf("coinbase tx %x has invalid recipient: %w", tx.Hash(), err)
 						}
-						if statedb.GetCode(internal) == nil || true {
```

Comment:
put a block number in so that we dont get this activated until then

Reply:
Done